wicamo96/FieldScout
# finished crud functionality for and displaying pests

# Description

Added CRUD functionality for pests and a page to display a list of all pests.

## Type of change


# Testing Instructions

### Create database and populate data if necessary
- If you haven't created the FieldScout database, please do so in visual studio community.
- Use this to fill out tables and columns: [SQL data](https://github.com/wicamo96/FieldScout/blob/main/SQL/01_FieldScout_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/wicamo96/FieldScout/blob/main/SQL/02_FieldScout_Seed_Data.sql)

### Testing Greenhouse Management CRUD
- Click the execute with debugger solid green button to launch the backend in visual studio community.
- From the client directory in a git terminal, run `npm run dev`.
- Once the Vite + React window pops up, you should be directed to the log in page.
- Log in with the default email.
- Navigate to the pests list via pest management in the navbar.
- Verify the list is populated with default pests (Aphids, Fungus Gnats, Spider Mites, Thrips, and Whiteflies).
- Add a test pest and verify that its added to the list.
- Test edit and delete functionality on your test pest.

# Checklist:
